Assignment2/TheDosah-fromRDFtoJSON-LD.jsonld

```diff
@@ -1,42 +1,58 @@
-      "@id":"http://www.upm.net/Class01",
-      "@type":"class:Class01",
-      "includes":[
-         {
-            "Sensor029":{
-               "@id":"http://www.upm.net/Class01/Sensor029",
-               "@type":"i:Sensor029",
-               "hasMeasurements":{
-                  "Measurements8401":{
-                     "@id":"http://www.upm.net/Class01/Sensor029/Measurements8401",
-                     "@type":"j:Measurements8401",
-                     "hasTemperature":{
-                        "@value":"29",
-                        "@type":"xsd:integer"
-                     },
-                     "atTime":{
-                        "@value":"2010-06-12T12:00:12",
-                        "@type":"xsd:dateTime"
-                     }
-                  }
-               }
-            }
-         },
-         {
-            "Computer101":{
-               "@id":"http://www.upm.net/Class01/Computer101",
-               "@type":"i:Computer101",
-               "hasOwner":{
-                  "User10A":{
-                     "@id":"http://www.upm.net/Class01/Computer101/User10A",
-                     "@type":"j:User10A",
-                     "hasName":{
-                        "@value":"Pedro",
-                        "@type":"xsd:string"
-                     }
-                  }
-               }
-            }
-         }
-      ]
-   }
-}
\ No newline at end of file
+{
+"@context": {
+	"vocab": "http://devices.com#",
+	"xsd": "http://www.w3.org/2001/XMLSchema#",
+	"Class01": "vocab:Class01",
+	"Computer101": "Class01:Computer101",
+	"Measurement8401": "Sensor029:Measurement8401",
+	"User10A": "Computer101:User10A",
+	"includes": "vocab:includes",
+	"hasMeasurement": "vocab:hasMeasurement",
+	"hasTemperature": "vocab:hasTemperature",
+	"atTime":"vocab:atTime",
+	"hasOwner": "vocab:hasOwner",
+	"hasName": "vocab:hasName"
+	
+},
+
+"@id":"http://www.upm.net/Class01",
+  "@type":"class:Class01",
+  "includes":[
+	 {
+		"Sensor029":{
```

Comment:
You don't need to define "Sensor029": because it isn't an attribute, it is a resource, thus I suggest you remove it. The Sensor09's content is ok, I mean leave the attributes you defined inside its curly braces